src/main/java/com/epam/ta/reportportal/entity/project/ProjectAnalyzerConfig.java

```diff
@@ -51,12 +39,13 @@ public ProjectAnalyzerConfig(int minDocFreq, int minTermFreq, int minShouldMatch
 		this.numberOfLogLines = numberOfLogLines;
 	}
 
-	public Long getId() {
-		return id;
-	}
-
-	public void setId(Long id) {
-		this.id = id;
+	public ProjectAnalyzerConfig(int minDocFreq, int minTermFreq, int minShouldMatch, int numberOfLogLines, boolean indexingRunning, Boolean isAutoAnalyzerEnabled) {
```

Comment:
@plisunov, sounds like it is good moment to refactor constructors in favor of builder. 

Regarding existing constructor `ProjectAnalyzerConfig(int minDocFreq, int minTermFreq, int minShouldMatch, int numberOfLogLines)` - I am not sure we should care of backward-compatibility here, so let's remove it.

Reply:
Give me Lombok and I will use Builder in many places  :-)
Regarding constructor - Done.

Reply:
Sorry, I don't see changes related to builder in this PR. Please, double-check.

Reply:
Done for ProjectAnalyzerConfigBuilder & ProjectEmailConfigBuilder